src/scripts/pokeDetail.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', async function () {
+  const params = new URLSearchParams(window.location.search);
+  const pokemonName = params.get('name');
+
+  const pokemonImage = document.getElementById('pokemonImage');
+  const pokemonNameElement = document.getElementById('pokemonName');
+  const pokemonAbilities = document.getElementById('pokemonAbilities');
+  const prevButton = document.getElementById('prevButton');
+  const nextButton = document.getElementById('nextButton');
+  const goToListButton = document.getElementById('goToListButton');
+
+  async function showPokemonData(pokemonName) {
+    try {
+      const response = await fetch(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`);
```

Comment:
@Kamildeeal please create a separate file for contants like `constants.ts` and put there `const BASE_API_URL = 'https://pokeapi.co/api'`. Then you'll be able to reuse it for all of your links, instead of repeating